src/vs/workbench/contrib/inlineChat/browser/inlineChatController.ts

```diff
@@ -594,19 +598,23 @@ export class InlineChatController implements IEditorContribution {
 		const { response } = this._activeSession.lastExchange!;
 		this._showWidget(false);
 
+		let status: string | undefined;
+
 		this._ctxLastResponseType.set(response instanceof EditResponse || response instanceof MarkdownResponse
 			? response.raw.type
 			: undefined);
 
 		if (response instanceof EmptyResponse) {
 			// show status message
-			this._zone.value.widget.updateStatus(localize('empty', "No results, please refine your input and try again"), { classes: ['warn'] });
+			status = localize('empty', "No results, please refine your input and try again");
+			this._zone.value.widget.updateStatus(status, { classes: ['warn'] });
 			return State.WAIT_FOR_INPUT;
 
 		} else if (response instanceof ErrorResponse) {
 			// show error
 			if (!response.isCancellation) {
-				this._zone.value.widget.updateStatus(response.message, { classes: ['error'] });
+				status = localize('errorResponseMessage', "{0}", response.message);
```

Comment:
`localize` doesn't do anything here because it's just the none-localized parameter (response.message)